sorbet-coerce.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.files         += Dir.glob('spec/**/*')
   s.files         += Dir.glob('rbi/**/*')
 
-  s.required_ruby_version = ['>= 2.3.0']
+  s.required_ruby_version = ['>= 2.3.0', '< 2.7.0']
```

Comment:
what's the error with 2.7.0?

Reply:
Hmm seems I wasn't able to get a newer sorbet version on 2.7. Then, we have:
```An error occurred while loading ./spec/nested_spec.rb.
Failure/Error: require 'sorbet-runtime'

FrozenError:
  can't modify frozen Module
# /usr/share/rvm/gems/ruby-2.7.0-preview1/gems/sorbet-runtime-0.4.5173/lib/types/private/types/void.rb:17:in `<class:Void>'
# /usr/share/rvm/gems/ruby-2.7.0-preview1/gems/sorbet-runtime-0.4.5173/lib/types/private/types/void.rb:6:in `<top (required)>'
# /usr/share/rvm/gems/ruby-2.7.0-preview1/gems/sorbet-runtime-0.4.5173/lib/sorbet-runtime.rb:58:in `require_relative'
# /usr/share/rvm/gems/ruby-2.7.0-preview1/gems/sorbet-runtime-0.4.5173/lib/sorbet-runtime.rb:58:in `<top (required)>'
# ./lib/private/converter.rb:3:in `require'
# ./lib/private/converter.rb:3:in `<top (required)>'
```

The frozen module in the stack trace:
```VOID = Module.new.freeze```